com.gluonhq.SceneBuilder.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.gluonhq.SceneBuilder
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder-runner
+rename-desktop-file: scenebuilder-SceneBuilder.desktop
+finish-args:
+  - --env=PATH=/app/bin
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --persist=.scenebuilder
+  - --filesystem=home
+cleanup:
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
```

Comment:
why do you delete the locales?

Reply:
I wasn't sure about this, took it from the original PR #3402
Will remove them now